src/cart/_structures_from_macros_.h

```diff
@@ -100,6 +100,23 @@ struct crt_ctl_get_pid_out {
 	int32_t cgp_rc;
 };
 
+struct crt_ctl_get_uri_cache_in {
+	crt_group_id_t cel_grp_id;
+	d_rank_t cel_rank;
+};
+
+struct crt_ctl_get_uri_cache_out {
+<<<<<<< HEAD
+	struct {
+	uint64_t ca_count;
+	struct crt_grp_cache *ca_arrays;
+	} cguc_grp_cache;
+=======
```

Comment:
(style)  spaces required around that '==' (ctx:ExO)

(style)  spaces required around that '==' (ctx:OxO)

(style)  spaces required around that '==' (ctx:OxO)

(style)  spaces required around that '=' (ctx:OxE)